DeltaCore/UI/Controller/ControllerView.swift

```diff
@@ -292,11 +293,18 @@ public extension ControllerView
         self.buttonsView.controllerSkin = self.controllerSkin
         self.buttonsView.controllerSkinTraits = self.controllerSkinTraits
         
+        var debugModeEnabled = false
         if let isDebugModeEnabled = self.controllerSkin?.isDebugModeEnabled
         {
-            self.controllerDebugView.isHidden = !isDebugModeEnabled
+            debugModeEnabled = isDebugModeEnabled
+        }
+        if self.isControllerSkinDebugModeEnabled == true
+        {
+            debugModeEnabled = self.isControllerSkinDebugModeEnabled
         }
         
+        self.controllerDebugView.isHidden = !debugModeEnabled
+        
```

Comment:
I plan to completely remove `ControllerSkin.isDebugModeEnabled`, so this logic can be simplified to only checking `self.isControllerSkinDebugModeEnabled`